website/docs/d/launch_template.html.markdown

```diff
@@ -18,11 +18,30 @@ data "aws_launch_template" "default" {
 }
 ```
 
+### Filter
+
+```hcl
+data "aws_launch_template" "test" {
+  filter {
+    name   = "launch-template-name"
+    values = ["some-template"]
+  }
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the launch template.
+* `name` - (Optional) The name of the launch template.
+* `filter` - (Optional) Configuration block(s) for filtering. Detailed below.
```

Comment:
Oh ha, missing `tags` argument here -- will fix on merge. 👍 

```suggestion
* `filter` - (Optional) Configuration block(s) for filtering. Detailed below.
* `name` - (Optional) The name of the launch template.
* `tags` - (Optional) A mapping of tags, each pair of which must exactly match a pair on the desired Launch Template.
```

Reply:
i think i may have missed this on a bunch of them after the last fix. will check